dockers/docker-dhcp-relay/isc-dhcp-relay

```diff
@@ -0,0 +1,6 @@
+SERVERS=""
+
+INTERFACES=""
+
+#-a provides option 82 circuit id information
```

Comment:
Yes, the -a option enables the use of DHCP Option 82. I think I may have accidently removed the OPTIONS="-a". I will fix this.